package/MDAnalysis/analysis/pca.py

```diff
@@ -391,14 +397,17 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
         .. versionchanged:: 0.19.0
            Transform now requires that :meth:`run` has been called before,
            otherwise a :exc:`ValueError` is raised.
+        .. versionadded:: 2.8.0
+           Transform now has shows a tqdm progressbar, which can be toggled
+           on with verbose = True, or off with verbose = False
```

Comment:
1. When adding to the changes _at the bottom_ of the class, this *must* be `.. versionchanged:: 2.8.0` because there's only one time that the whole class was added. (Alternatively one can add versionchanged/added for individual blocks – see docs on numpy docs/sphinx). The way you're doing is her is fine, just change to versionchanged.
2. you can use markup
   ```reST
   on with ``verbose = True``, or off with ``verbose = False``
   ```